site/content/en/docs/drivers/includes/none_usage.inc

```diff
@@ -2,6 +2,8 @@
 
 VM running a systemd-based Linux distribution ([see #2704](https://github.com/kubernetes/minikube/issues/2704))
```

Comment:
while we are at this, anywhere in the documentation we say it required systemd needs to be cahnged, thomas added a feature that it works on openRC too.

(I believe there is 2-3 places we ask ppl to verify they run on systemd, that all needs to be corrected)

Reply:
@medyagh - I created a separate issue to fix that https://github.com/kubernetes/minikube/issues/7651

Reply:
I can fix that issue in a separate PR.

Reply:
Ok